app/controllers/rents_controller.rb

```diff
@@ -0,0 +1,18 @@
+class RentsController < ApplicationController
+  include Wor::Paginate
+
+  def create
+    @rent = Rent.create(rent_create_params)
+    render json: @rent, each_serializer: RentSerializer
```

Comment:
what about the status code here? Which status code are you returning for this create?